srtcore/channel.cpp

```diff
@@ -440,45 +678,23 @@ int CChannel::sendto(const sockaddr* addr, CPacket& packet) const
    return res;
 }
 
-EReadStatus CChannel::recvfrom(sockaddr* addr, CPacket& packet) const
+EReadStatus CChannel::sys_recvmsg(ref_t<CPacket> r_pkt, ref_t<int> r_result, ref_t<int> r_msg_flags, sockaddr* addr) const
```

Comment:
Do we really need those `ref_t` staff here? Are they just to highlight that the argument is passed by reference?

Reply:
Yes. And this should have virtually no overhead, it just wraps a single pointer.